packages/radio/src/react/button.js

```diff
@@ -28,86 +28,80 @@ const styles = {
     )
 }
 
-const Button = React.forwardRef((props, forwardedRef) => {
-  const themeName = useTheme()
+const isChecked = (a, b) => a === b
 
+const Button = React.forwardRef(({ value, label, ...props }, forwardedRef) => {
+  const themeName = useTheme()
+  const { checkedValue, onSelect, disabled, error, name } = useRadioContext()
   const ref = React.useRef()
   React.useImperativeHandle(forwardedRef, () => ref.current)
 
```

Comment:
you probably wanted to calculate `checked` doing something like

```
const checked = checkedValue === value
```

Reply:
oh...looks like you have an isChecked function. so i suggest using that once here to have a single variable with the boolean. and then making sure that's used everywhere - right now the functionality is broken because of the `checked` coming from glamor

```
const checked = isChecked(value, checkedValue)
```

Reply:
why?